### DISCUSSION TITLE
tide_correction.csh error

### DISCUSSION THREAD

**Discussion by sreeram-radhakrishnan** [gmtsar/gmtsar#806] [Q&A]
Hello everyone, I have got some error while running tide_correction.csh . It was showing warning and other error, then i tried to run the tide.grd into phasefilt.grd for subtraction, I'm getting error as grd files are not same size. How to solve this issue? Help me. I have attached the error message of tide correction. Thank you.
![Screenshot_2023-09-08_12-43-44](https://github.com/gmtsar/gmtsar/assets/85307395/c40eed8f-491e-42d2-944b-20b9ac26ddb3)

**Comment by Xiaohua-Eric-Xu** [comment]
Please check my responses here: https://github.com/gmtsar/gmtsar/issues/805
